multi-arch.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
```

Comment:
Could you also add one for aarch64?

Reply:
I had an aarch64 version of this, but not really sure it would be suitable since this is targeted to beaglebone blue. But I can add one anyways.